modules/wss4j/src/org/apache/ws/security/saml/SAMLIssuerImpl.java

```diff
@@ -31,20 +31,16 @@
 import org.apache.xml.security.keys.content.keyvalues.DSAKeyValue;
 import org.apache.xml.security.keys.content.keyvalues.RSAKeyValue;
 import org.apache.xml.security.signature.XMLSignature;
-import org.opensaml.SAMLAssertion;
-import org.opensaml.SAMLAuthenticationStatement;
-import org.opensaml.SAMLException;
-import org.opensaml.SAMLNameIdentifier;
-import org.opensaml.SAMLStatement;
-import org.opensaml.SAMLSubject;
+
+import org.joda.time.DateTime;
+import org.opensaml.saml.saml1.core.*;
```

Comment:
avoid * imports

Reply:
fixed in 2f696f6746f53ec59ed149ea7753dbcb09f7b044